proto/quicksilver/participationrewards/v1/participationrewards.proto

```diff
@@ -48,7 +49,7 @@ message KeyedProtocolData {
 // determination of rewards.
 message ProtocolData {
   string type = 1;
-  bytes data = 2 [(gogoproto.casttype) = "encoding/json.RawMessage"];
+  bytes data = 2 [ (gogoproto.casttype) = "encoding/json.RawMessage" ];
```

Comment:
can you try as alex advised, and run `make proto-setup` which should update the version of buf we are using and hopefully avoid the constant proto formatting changes. Thanks :)

Reply:
I did that, still no difference

Reply:
Ah - I suspect you need to run `make proto-format`. Might be worth making this automatically trigger after `make proto-gen`

Reply:
done